src/univalue/univalue.h

```diff
@@ -154,7 +154,8 @@ class UniValue {
     bool push_back(std::pair<std::string,UniValue> pear) {
         return pushKV(pear.first, pear.second);
     }
-    friend const UniValue& find_value( const UniValue& obj, const std::string& name);
+    //!return a value if key was found (only supports VOBJ types)
```

Comment:
I would say this comment is appropriate. The keys vector is only >0 for VOBJ objects.